src/odin/enhancedtrippath.cc

```diff
@@ -1298,6 +1430,72 @@ uint32_t EnhancedTripLeg_Node::GetStraightestIntersectingEdgeTurnDegree(uint32_t
   return staightest_turn_degree;
 }
 
+uint32_t EnhancedTripLeg_Node::GetRightMostTurnDegree(uint32_t turn_degree,
+                                                      uint32_t from_heading,
+                                                      const TripLeg_TravelMode travel_mode) {
+
+  auto get_right_delta = [](uint32_t turn_degree) -> uint32_t {
+    if (turn_degree < 90) {
+      return (90 - turn_degree);
+    } else if (turn_degree > 270) {
+      return (360 - turn_degree + 90);
+    } else {
+      return (turn_degree - 90);
+    }
+  };
+
+  uint32_t right_most_turn_degree = turn_degree;
+  uint32_t right_most_delta = get_right_delta(turn_degree);
+
+  for (int i = 0; i < intersecting_edge_size(); ++i) {
+    // Only process traversable outbound edges
+    if (GetIntersectingEdge(i)->IsTraversableOutbound(travel_mode)) {
+      // Get the intersecting edge turn degree and right turn delta
+      uint32_t xturn_degree = GetTurnDegree(from_heading, intersecting_edge(i).begin_heading());
+      uint32_t right_delta = get_right_delta(xturn_degree);
+      // Determine if the intersecting edge turn degree is closer to true right (90)
+      if (right_delta < right_most_delta) {
+        right_most_delta = right_delta;
+        right_most_turn_degree = xturn_degree;
+      }
+    }
+  }
+  return right_most_turn_degree;
+}
+
+uint32_t EnhancedTripLeg_Node::GetLeftMostTurnDegree(uint32_t turn_degree,
```

Comment:
will replace constants with this [issue](https://github.com/valhalla/valhalla/issues/1739)